doc/api/cli.md

```diff
@@ -1980,6 +1980,24 @@ On a machine with 2 GB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max_semi_space_size=SIZE` (in megabytes)
+
+Sets the maximum
+[semi\_space](https://www.memorymanagement.org/glossary/s.html#semi.space)
```

Comment:
memorymanagement.org uses `semi-space`, V8 docs uses `semispace`. The V8 `--help` also uses `semi-space`, we should probably stick with that.
```suggestion
[semi-space](https://www.memorymanagement.org/glossary/s.html#semi.space)
```